Packs/CommonScripts/ReleaseNotes/1_6_30.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### New: ParseEmailFilesV2
+- The script will be in beta for 3 months And then it will replace the *ParseEmailFiles* script.
+- Parse an email from an eml or msg file and populate all relevant context data to investigate the email.
```

Comment:
```suggestion
- Parse an email from an eml or msg file using the parse-emails XSOAR python package, and populate relevant fields and data into an XSOAR investigation. 
- The script is being released in Beta mode, after 3 months it will be GA and will officially replace the *ParseEmailFiles* script.
```